web/static/js/components/idea_board.jsx

```diff
@@ -8,7 +8,7 @@ import styles from "./css_modules/idea_board.css"
 const IdeaBoard = props => {
   const { stage } = props
   const categories = ["happy", "sad", "confused"]
-  const showActionItem = stage !== "idea-generation"
+  const showActionItem = !["idea-generation", "voting"].includes(stage)
```

Comment:
to avoid the `!` here, we could just say that we only want to show the action items during particular stages, rather than list the stages that we don't want, i.e. `["action-items", "closed"].includes(stage)`.

In the long term we want to extract these conditionals and have composed elements like `IdeaGenerationIdeaboard` and `ActionItemIdeaBoard`, as these in-component conditionals are only going to get more complex as we add stages.